### PR TITLE
MON-75910-telegraf

### PR DESCRIPTION
**Pull request by fmattesct**
## Description

Added a new documentation page for Telegraf support in Centreon.
Beta scope.

## Target version (i.e. version that this PR changes)

- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/getting-started/how-to-guides/telegraf.md [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+</Tabs>
+
+
+
+#### Windows
+
+Téléchargez et exécutez le package pour [Windows](https://github.com/centreon/centreon-nsclient-build/releases/download/20240325/centreon_plugins.exe)
+
+
+### Créer les équipements en utilisant les modèles
```
Comment (fmattesct) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/getting-started/how-to-guides/telegraf.md:280]:
```suggestion

Créer les équipements en utilisant les modèles
```

File: pp/integrations/plugin-packs/getting-started/how-to-guides/telegraf.md [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+</Tabs>
+
+
+
+#### Windows
+
+Download and execute package for [Windows](https://github.com/centreon/centreon-nsclient-build/releases/download/20240325/centreon_plugins.exe)
+
+
+### Create hosts using templates
```
Comment (fmattesct) [inline pp/integrations/plugin-packs/getting-started/how-to-guides/telegraf.md:281]:
```suggestion

Create hosts using templates
```

**Review by fmattesct — COMMENTED** [review]
Changed last title format

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/getting-started/how-to-guides/telegraf.md [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+
+![image](../../../../assets/integrations/plugin-packs/how-to-guides/telegraf-connector.png)
+
+Créez un nouveau connecteur avec les valeurs suivantes :
+
+* Nom du connecteur : Telegraf
+* Description du connecteur : Telegraf 
+* Ligne de Commande :
+~~~
+open_telemetry --processor=nagios_telegraf --extractor=attributes --host_path=resource_metrics.scope_metrics.data.data_points.attributes.host --service_path=resource_metrics.scope_metrics.data.data_points.attributes.service
```
Comment (jean-christophe81) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/getting-started/how-to-guides/telegraf.md:58]: opentelemetry

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/getting-started/how-to-guides/telegraf.md [hunk 4]
```diff
@@ -0,0 +1,280 @@
...
+~~~
+**open_telemetry** indiques à Engine que ce n'est pas un vrai connecteur
```
Comment (jean-christophe81) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/getting-started/how-to-guides/telegraf.md:60]: idem

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/getting-started/how-to-guides/telegraf.md [hunk 5]
```diff
@@ -0,0 +1,280 @@
...
+ "max_length_grpc_log": 0,
+ "telegraf_conf_server": {
+   "http_server": {
+     "port": 1443,
+     "encryption": true,
+     "certificate_path": "/tmp/otel/server.crt",
+     "key_path": "/tmp/otel/server.key"
+   },
+   "telegraf_conf": {
+     "interval": "10s",
```
Comment (jean-christophe81) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/getting-started/how-to-guides/telegraf.md:99]: 60s

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/getting-started/how-to-guides/telegraf.md [hunk 6]
```diff
@@ -0,0 +1,280 @@
...
+* Description du connecteur : Telegraf 
+* Ligne de Commande :
+~~~
+open_telemetry --processor=nagios_telegraf --extractor=attributes --host_path=resource_metrics.scope_metrics.data.data_points.attributes.host --service_path=resource_metrics.scope_metrics.data.data_points.attributes.service
+~~~
+**open_telemetry** indiques à Engine que ce n'est pas un vrai connecteur
+
+**--processor=nagios_telegraf**    
+Toutes les données reçues sont stockées dans des files FiFo.  
+Ce processeur affichera ces données afin de générer des résultats de vérification
```
Comment (jean-christophe81) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/getting-started/how-to-guides/telegraf.md:64]: Ce processeur utilisera ces données afin de générer des résultats de vérification

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/assets/integrations/plugin-packs/how-to-guides/telegraf-connector.png
Comment (jean-christophe81) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/assets/integrations/plugin-packs/how-to-guides/telegraf-connector.png:1]: opentelemetry, not open_telemetry

File: pp/assets/integrations/plugin-packs/how-to-guides/telegraf-connector.png
Comment (jean-christophe81) [inline pp/assets/integrations/plugin-packs/how-to-guides/telegraf-connector.png:1]: opentelemetry, not open_telemetry

**Review by jean-christophe81 — APPROVED** [review]

**Review by cg-tw — APPROVED** [review]

**Pull request merged**